src/msg/async/rdma/Infiniband.cc

```diff
@@ -567,10 +566,8 @@ Infiniband::MemoryManager::Cluster::Cluster(MemoryManager& m, uint32_t s)
 
 Infiniband::MemoryManager::Cluster::~Cluster()
 {
-  const auto chunk_end = chunk_base + num_chunk;
-  for (auto chunk = chunk_base; chunk != chunk_end; chunk++) {
-    chunk->~Chunk();
-  }
```

Comment:
I think we could save this. if we add logic to Chunk later, this will preserve we forget to deconstruct